lib/std/http/method.zig

```diff
@@ -14,4 +14,87 @@ pub const Method = enum {
     OPTIONS,
     TRACE,
     PATCH,
+
+    /// Returns true if a request of this method is allowed to have a body
+    /// Returns null if it also depends on other conditions
+    pub fn requestHasBody(self: Method) ?bool {
+        return switch (self) {
+            .GET => false,
+            .HEAD => false,
+            .POST => true,
+            .PUT => true,
+            .DELETE => null,
+            .CONNECT => false,
```

Comment:
What do you call the contents of a CONNECT if not a body?

Reply:
idk, I would return `null`, since it seems similar to ws-upgrade, I wouldn't count that as body, because it's a stream I suppose...

Reply:
its about *shouldn't*, something the client/server implementation can use in debug mode, the info is from MDN

Reply:
gonna rework the doc comment to reflect this